development/en-us/e2e-test.md

```diff
@@ -177,9 +177,15 @@ The rest are similar cases and can be understood by referring to the specific so
 https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases
 
 ##  III. Supplements
+When running E2E tests locally, First, you need to start the local service, you can refer to this page: 
```

Comment:
Could you add one blank line after title?

Reply:
Same as zh docs.

Reply:
I'' add it.